Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -283,3 +294,17 @@ const std::string GameListItem::GetWiiFSPath() const
 	return ret;
 }
 
+bool GameListItem::IsElfOrDol() const
```

Comment:
To truly make this more robust you would do:

```cpp
bool GameListItem::IsElfOrDoll() const
{
	const std::string name = GetName();
	const size_t pos = name.rfind('.');

	if (pos != std::string::npos)
	{
		std::string ext = name.substr(pos);
		std::transform(ext.begin(), ext.end(), ext.begin(), ::tolower);
		
		return ext == ".elf" ||
		       ext == ".dol";
	}
	return false;
}
```

or convert it to a wxString and call ToLower on it.